src/main/java/net/neoforged/neoforge/client/model/EmptyModel.java

```diff
@@ -51,8 +54,17 @@ public BakedModel bake(IGeometryBakingContext context, ModelBaker baker, Functio
     private static class Baked extends SimpleBakedModel {
         private static final Material MISSING_TEXTURE = new Material(TextureAtlas.LOCATION_BLOCKS, MissingTextureAtlasSprite.getLocation());
 
+        // SimpleBakedModel must have a quad list per face in its map.
+        private static Map<Direction, List<BakedQuad>> makeEmptyCulledFaces() {
```

Comment:
This should be wrapped in `Collections.unmodifiableMap` and stored as a static constant, IMO.

Reply:
The model is a singleton. And vanilla doesn't use unmodifiable maps for that.

Reply:
Oh I'm blind, missed that it was a singleton.